get_web_content.pl

```diff
@@ -37,7 +38,14 @@
 
     # Get username and password from authorization file(s) (which might be a glob)
     if (defined $web_opt{auth_file}) {
-        ($web_opt{user}, $web_opt{passwd}) = Ska::Web::get_user_passwd($web_opt{auth_file})
+	if ($web_opt{auth_file} eq 'occweb'){
+	    my $netrc = Net::Netrc->lookup('occweb');
+	    $web_opt{user} = $netrc->login;
+	    $web_opt{passwd} = $netrc->password;
+	}
+	else{
+	    ($web_opt{user}, $web_opt{passwd}) = Ska::Web::get_user_passwd($web_opt{auth_file})
```

Comment:
Why is this still supported? It looks like it is orphaned given the update to `web_content.cfg`, and this flexibility is not in `get_iFOT_events`. We could just get rid of the `auth_file` option entirely?

Reply:
So, I tested that the I could run get_iFOT_events and get_web_content with the current profile.  Checking that nothing else in here was using those pieces and configurables seemed more steps requiring more verification.  If you want that, that's fine too.

Reply:
With regard to removing auth_file completely, I suppose that would be a different exercise... I think you'd need to special case occweb in the calling/fetching code if you no longer had something in the config, otherwise you'd send incorrect occweb credentials when doing every other web query.

Reply:
Mostly you are just seeing that I was trying to make changes with the smallest scope and possibility for error.

Reply:
If we just care about the current task schedule use case being tested (what I did) then it is fine to clean this up and remove the support of additional/alternative auth_file configurations.  I just replated the web_content.cfg entry with a new netrc keyword (can be renamed as you like) and removed the conditional in here.